interface/forms/gad7/report.php

```diff
@@ -12,19 +13,33 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+/* these are included in gad7.inc.php
 require_once(dirname(__FILE__) . '/../../globals.php');
 require_once($GLOBALS["srcdir"] . "/api.inc");
+*/
+require_once("gad7.inc.php");
+use Mpdf\Mpdf;  /* used to generate a pdf of the form */
```

Comment:
place a newline between the `require_once` block and the `use` block

Reply:
actually this import is not needed and can be removed.